Logging/LogHandler/FileLogHandler.cs

```diff
@@ -56,14 +73,58 @@ public class FileLogHandler : AbstractAnvilBase, ILogHandler
         /// </summary>
         public LogLevel MinimumLevel { get; set; } = LogLevel.Debug;
 
+        /// <summary>
+        /// The maximum file size in bytes before the file is rotated. Default is 10MB.
+        /// </summary>
+        public long RotateFileSizeLimit { get; set; } = 10L * 1024 * 1024;
+
+        /// <summary>
+        /// The maximum number of rotated files to keep before deleting the oldest. Default is 5.
+        /// Ex. If the limit is 5, up to 5 rotated files in addition to the original log file, will be kept. After reaching this limit,
+        /// the next time the log file is rotated, the oldest file will be deleted. If the limit is 0, no rotated files are kept,
+        /// i.e. when the log file is rotated, it is simply erased and starts over.
+        /// </summary>
+        public int RotateFileCountLimit { get; set; } = 5;
+
```

Comment:
I'm not sure where but should we add some sort of `<remarks>` comment about how this work similar to the Unix rotate thing @mbaker3 was talking about in #108 ?

Reply:
I ended up defining "file rotation" in the constructor under the `rotate` param, but let me know if there's a better way to document that.